### DISCUSSION TITLE
Mobile android/ios app

### DISCUSSION THREAD

**Discussion by Zorg64** [seerr-team/seerr#340] [Q&A] [answered]
Sorry if this is posted somewhere and I just missed it.

Is there a dedicated app for mobile and/or is it planned?

**Comment by fallenbagel** [comment, marked as answer]
Jellyseerr is a pwa(progressive web app). Meaning you can install it as a normal app. If you go to the website (has to be https) on your android let's say for example chrome, you can press the three dots and select install and it will install as a normal app like so:

![Screenshot_20230306_003817_One UI Home](https://user-images.githubusercontent.com/98979876/222981923-b62153b0-a197-4760-a5af-607411ca197b.jpg)

As for iphone, you can press add to home screen to install it after pressing share.

Pwa supports Web push notifications so you can turn them on in notifications and have notifications like a native app.

**Reply by Zorg64** [reply]
Thank you for the quick reply. 

Might I suggest documenting this, and if it already is maybe making it a bit more apparent? As I (and also likely, others) wasn't even going to give it a go until there was an app with notifications.

I did see in features:

```
- Support for various notification agents.
- Mobile-friendly design, for when you need to approve requests on the go!
```

Though thought that: 'various notifications agents' referred to such things like, discord, pushbullet, etc... not a 'self-contained' mobile app, non 3rd party notifications.  And 'mobile-friendly design', just meant that it had a mobile-friendly webpage. 

Thanks again.

**Reply by fallenbagel** [reply]
Will do. Thanks for the suggestion!

**Reply by Hamburgalarr** [reply]
Sorry I know it has to be HTTPS but all my truenas scale apps access through http? how can I change this?

**Reply by fallenbagel** [reply]
> Sorry I know it has to be HTTPS but all my truenas scale apps access through http? how can I change this?

You'll need to add a reverse proxy and add an ssl cert to it. You can look at our reverse proxy docs. I would recommend caddy as its the easiest to setup. You'll also need a domain. You could use duckdns which is a free ddns as well

**Reply by ErraticFox** [reply]
Kind of crummy considering some people don't like home screen clutter and would much rather use an app drawer on Android. But with PWAs, they're only able to be saved to the home screen.

**Reply by fallenbagel** [reply]
> Kind of crummy considering some people don't like home screen clutter and would much rather use an app drawer on Android. But with PWAs, they're only able to be saved to the home screen. 

That depends on browser. Mine stores in app drawer. Not on home screen. 

And if you dont want to use a pwa, then wait for the app i guess or use it as it currently is.

**Reply by ErraticFox** [reply]
So I messed with PWAs before they had the ability to "install" (like in Chrome on Android). And I've recently moved over to FireFox which doesn't support this. So when I tried the PWA through FireFox it put it on the home screen, I just assumed it was still the same regardless of what browser you used. 

That's nice to know. I just used Chrome to install it and we're good now. 🤝🎉
